listenbrainz/webserver/views/api_tools.py

```diff
@@ -263,6 +274,16 @@ def log_raise_400(msg, data=""):
     raise APIBadRequest(msg)
 
 
+def validate_duration_field(listen, key):
+    if key in listen['track_metadata']['additional_info']:
```

Comment:
Hi! I, @alastair and @mayhem discussed it in IRC. We think its better to throw a Validation error if both duration and duration_ms are present. Also, if 1 is present and its value is not a positive integer we should throw a ValidationError,

Reply:
Hi, Lucifer. Thanks for the advice. I modified the validation part and add new commits.